gatsby-ssr.js

```diff
@@ -1,5 +1,19 @@
 /* eslint-env node */
+const React = require('react')
 
 const PageWrapper = require('./src/components/organisms/PageWrapper').default
 
+const onRenderBody = ({ setHeadComponents }) => {
+  return setHeadComponents([
+    <script
+      key="plausible-custom-events"
+      dangerouslySetInnerHTML={{
+        __html:
+          'window.plausible = window.plausible || function() { (window.plausible.q = window.plausible.q || []).push(arguments) }'
+      }}
+    />
+  ])
+}
+
 exports.wrapPageElement = PageWrapper
+exports.onRenderBody = onRenderBody
```

Comment:
Is there anything preventing this from being in something like a `react-helmet` component instead of engaging the `gatsby-ssr` hooks? Not quite a blocker, just asking since Helmet is generally the happy path for these kinds of tasks.

Reply:
I attempted to place the script inside the `Helmet` component, but it caused an error:

![image](https://user-images.githubusercontent.com/43496356/146445752-fb315fa4-bb15-4f63-aed8-42c991d02b2f.png)

![image](https://user-images.githubusercontent.com/43496356/146445812-2cbf5fc2-dfc0-4c5d-95c8-6a39acb30864.png)

But if you know of a way to use the script inside `Helmet`, we can definitely do that!